{"name": "test_failed_login", "status": "passed", "description": "../fixtures/login.feature: Failed login with invalid credentials", "start": 1759200287611, "stop": 1759200287974, "uuid": "daa11eef-49c1-4608-9041-6d98a5bf4f38", "historyId": "eb98303b78b0dd8a81a7d4c9688110a7", "testCaseId": "eb98303b78b0dd8a81a7d4c9688110a7", "fullName": "tests.test_login#test_failed_login", "labels": [{"name": "feature", "value": "Login flow"}, {"name": "story", "value": "2.Failed login"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "MacBookPro"}, {"name": "thread", "value": "10086-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}], "titlePath": ["tests", "test_login.py"]}